app/app.rb

```diff
@@ -0,0 +1,79 @@
+ENV['RACK_ENV'] ||= 'development'
+
+require 'sinatra/base'
+require './app/data_mapper_setup'
+require 'bcrypt'
+require 'sinatra/flash'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+
+	get '/links' do
+		@links=Message.all
+		erb(:'links/index')
+	end
+
+  post '/links' do
+  link = Message.new(message: params[:message])
+  link.save
+  redirect to('/links')
+end
```

Comment:
Lint/BlockAlignment: `end` at 23, 0 is not aligned with `post '/links' do` at 19, 2.